src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -207,6 +224,7 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
   // List of people requested if group property is provided
   private _groupPeople: IDynamicPerson[];
   private _debouncedSearch: { (): void; (): void };
+  private _isFocused = false;
```

Comment:
I just discovered @internalProperty() and it can help here and you won't have to call requestUpdate below.

```suggestion
  @internalProperty() private _isFocused = false;
```

Reply:
great idea, works well! 

3eb164c